src/components/Header/index.jsx

```diff
@@ -27,28 +30,35 @@ import {
 } from "react-bootstrap";
 import { QuestionCircleFill, PersonCircle } from "react-bootstrap-icons";
 
-import Image from "../Widgets/Image";
-import routes from "../../constants/routes";
-import { logout, isAuth, getUserName, isAdmin } from "../../shared/authHelper";
+// Widgets
+import { Image } from "../Widgets";
```

Comment:
Shouldn't the following format be used as per the style guide?

```suggestion
import Image from "../Widgets";
```

Reply:
@GMishx Fixed it!